presto-main/src/test/java/com/facebook/presto/execution/executor/SimulationSplit.java

```diff
@@ -169,13 +138,13 @@ static class LeafSplit
     {
         private final long perQuantaNanos;
 
-        public LeafSplit(SimulationTask task, long scheduledTimeNanos, long perQuantaNanos)
+        LeafSplit(SimulationTask task, long scheduledTimeNanos, long perQuantaNanos)
```

Comment:
Move these to be after the public method getInfo()

Reply:
constructors usually come first